packages/css-selector-parser/src/ast-tools/specificity.ts

```diff
@@ -0,0 +1,105 @@
+import type {
+  ImmutableSelectorNode,
+  ImmutablePseudoClass,
+  SelectorNode,
+} from "../ast-types";
+import { walk } from "./walk";
+
+type Specificity = [number, number, number, number];
```

Comment:
give these and name [name: number, .....]

Reply:
sure